test/check-machines-create

```diff
@@ -1057,8 +1060,8 @@ vnc_password= "{vnc_passwd}"
                 query_result = f'{query_result}'
                 # check if there is os present in osinfo-query because it can be filtered out in the UI
                 # throws exception if grep fails
-                self.machine.execute(
-                    fr"osinfo-query os --fields=name | tail -n +3 | sed -e 's/\s*|\s*/|/g; s/^\s*//g; s/\s*$//g' | grep '{query_result}'")
+                sanitize_cmd = r"sed -e 's/\s*|\s*/|/g; s/^\s*//g; s/\s*$//g'"
+                m.execute(fr"osinfo-query os --fields=name | tail -n +3 | {sanitize_cmd} | grep '{query_result}'")
```

Comment:
This is ok, but I'd actually find it more readable to split the whole command into multiple lines at the `|`:
```
m.execute("cmd 1 |"
          "cmd 2 |" ..)
```

Reply:
You want me to do this in a follow up?

Reply:
Up to you, not that important.